code/datums/abnormality/datum/abnormality.dm

```diff
@@ -65,6 +68,8 @@
 	current.toggle_ai(AI_OFF)
 	current.status_flags |= GODMODE
 	current.setDir(EAST)
+	if (understanding == max_understanding)
```

Comment:
It is recommended to use >= instead of == here, just in case something does go wrong(i.e. it registers 2 works instead of 1, for any reason).

Reply:
The works are always Clamped though, they physically can't go above max_understanding.

Reply:
If there was any chance they could be above max_understanding I would, but the only way that's possible is direct admin-bussing to set it to above max_understanding